.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: ruby
```

Comment:
I don't think this file should be being added? I'd previously removed a travis config because it was breaking CI.

 Is there a reason to add it back in or is this just a merging problem?

Reply:
It was a merge, I just did the minimum and commented out the failing task to get the build to pass, Craig did suggest we could just sack the whole file.